pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -336,6 +336,9 @@ func (r *MXJobReconciler) UpdateJobStatus(job interface{}, replicas map[commonv1
 		}
 	}
 
+	//check whether mxnet Single Host training
+	singleTraining := r.onlyHasReplicaTypeWorker(replicas)
```

Comment:
singleHost

Reply:
Please update your comment to a complete sentence

Reply:
done